.evergreen/run-kms-servers.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
+set -o errexit  # Exit the script with error if any of the commands fail
 
 cd ${DRIVERS_TOOLS}/.evergreen/csfle
-. ./activate-kmstlsvenv.sh
-# by default it always runs on port 5698
-python -u kms_kmip_server.py &
-python -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/expired.pem --port 8000  &
-python -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/wrong-host.pem --port 8001  &
-python -u kms_http_server.py --ca_file ../x509gen/ca.pem --cert_file ../x509gen/server.pem --port 8002 --require_client_cert &
+if [ "Windows_NT" = "$OS" ]; then
+  PYTHON="kmstlsvenv/Scripts/python.exe"
+else
+  PYTHON="./kmstlsvenv/bin/python3"
+fi
```

Comment:
Why do we duplicate this logic across `run-kmip-server.sh` and `run-kms-servers.sh` instead of exporting this in `activate-kms-venv.sh` or a similarly named script?

Reply:
Moved into single spot in activate-kms-venv.sh